database/seeds/UsersSeeder.php

```diff
@@ -60,5 +60,8 @@ public function run()
 
         // So we reliably have one user who can act on behalf of all traders.
         $users[3]->traders()->sync($traders);
+
+        // So we reliably have a specific user who cannot access the tap page
+        $users[1]->traders()->sync($traders[6]);
```

Comment:
@charlesstrange2 this is not fully clear to me - I wanted my specific trader to be assigned to a specific user.

Reply:
the relationship of Users to Traders is Many to Many
so $users[1] gets a user off the array
... you then pull the users' traders() relationship 
... and sync() an array of traders ids to it.
... in this case 1 of them 

sync manages the pivot table (trader_users in the DB) that sits between the "user M>--<M trader" relationship.